src/components/dropdown-menu/index.js

```diff
@@ -1,3 +1,3 @@
 // @flow
-export { default } from './DropdownMenu';
+export { default, ResponsiveDropdownMenu } from './DropdownMenu';
```

Comment:
Can we move this to a separate component barrel file to avoid introducing a new dependency?